gravitee-am-repository/gravitee-am-repository-mongodb/src/main/java/io/gravitee/am/repository/mongodb/management/internal/model/EnrollSettingsMongo.java

```diff
@@ -32,6 +32,8 @@ public class EnrollSettingsMongo {
     private Boolean forceEnrollment;
     private Long skipTimeSeconds;
     private String enrollmentRule;
+    private Boolean enrollmentSkipActive;
+    private String enrollmentSkipRule;
```

Comment:
**suggestion**: I think we can reduce the attribute name as we are in the EnrollSetting object. wdyt ? This apply to all places where these attributes has been added (UI, Bean and this Mongo bean)
 
```suggestion
    private Boolean skipActive;
    private String skipRule;
```

Reply:
I want to align with "enrollmentRule" to tell that it can skip the "enrollmentRule". If we want to reduce the name we should do it also for "enrollmentRule" and update it to "rule".

Reply:
understand but as we have two rule attributes in this settings, I'm not chocked by the fact that one is named enrollmentRule and teh other skipRule. I let you decide what you consider the best option in term of readability.